src/main/services/outputProcessors/soot/README.md

```diff
@@ -3,3 +3,22 @@
 1. Generate the conflict-static-analysis executable following the instructions on its README
 2. Move the generated file at `target/soot-analysis-<VERSION>-jar-with-dependencies.jar` to the directory `dependencies/` at miningframework
 3. Update the version passed to SootAnalysisWrapper at the `services.outputProcessors.soot.RunSootAnalysisOutputProcessor` so that it can resolve the file `dependencies/soot-analysis-<VERSION>-jar-with-dependencies.jar`
+
+
+## Run multiple scenarios
+
+1. Add an example of the file the [results-with-build-information.csv](../../../../../results-with-build-information.csv) into the `output/data` folder. This file contains the following fields:
```

Comment:
não deveria ser input?

Add a file to the ... folder, with the dataset information, that is, information about the merge scenarios to be analyzed, following the template of the example file .... . This file contains the following columns:

Reply:
output mesmo

Generate (by running the mining framework) or Add a file to the ... folder